source/includes/_08-additional-features.md.erb

```diff
@@ -369,8 +377,69 @@ app.post("/search", (req, res) => {
         An object uniquely identifying this candidate from this provider,
         structured as it is in <a href="#data-structure">the PassFort API</a>
       </td>
+    </tr>
     <tr>
-  </tbody>
+      <td><code>addresses</code></td>
+      <td>
+        A list of CompanyAddress objects
+      </td>
+      <td>No</td>
+      <td>
+        Addresses of the company, with each CompanyAddress object structured
+        as it is in <a href="#data-structure">the PassFort API</a>.
+      </td>
+    </tr>
+    <tr>
+      <td><code>contact</code></td>
+      <td>
+        A CompanyContactDetails object
+      </td>
+      <td>No</td>
+      <td>
+        Contact details for the company, in an object structured as it
+        is in <a href="#data-structure">the PassFort API</a>.
+      </td>
+    </tr>
+    <tr>
+      <td><code>incorporation_date</code></td>
+      <td>
+        string
+      </td>
+      <td>No</td>
+      <td>
+        Exact date in YYYY-MM-DD format
+      </td>
+    </tr>
+    <tr>
+      <td><code>structure_type</code></td>
+      <td>
+        Object containing structured fields
+      </td>
+      <td>No</td>
+      <td>
+        Object structured the same as the `structured_company_type` field of
+        the `CompanyMetadata` object in <a href="#data-structure">the PassFort
+        API</a>: "Structured fields indicating the kind of liability its
+        members are exposed to, the ownership structure, and whether it is
+        publically traded."
+      </td>
+    </tr>
+    <tr>
+      <td><code>lei</code></td>
+      <td>string</td>
+      <td>No</td>
+      <td>Global Legal Entity Identifier</td>
+    </tr>
+    <tr>
+      <td><code>tax_ids</code></td>
+      <td>List of TaxId objects</td>
+      <td>No</td>
+      <td>
+        Tax identifiers; each TaxId object is structured as in <a
+        href="#data-structure">the PassFort API</a>
```

Comment:
missing full-stop (it seems like the description fields have full-stops but not the type fields